mailbox/api/src/main/java/org/apache/james/mailbox/model/ParsedAttachment.java

```diff
@@ -20,12 +20,8 @@
 package org.apache.james.mailbox.model;
```

Comment:
step #8 is misleading: commit message has apparently nothing to do with what it changes